src/plugins/dashboard/tsconfig.json

```diff
@@ -24,6 +24,7 @@
     { "path": "../navigation/tsconfig.json" },
     { "path": "../saved_objects_tagging_oss/tsconfig.json" },
     { "path": "../saved_objects/tsconfig.json" },
+    { "path": "../screenshot_mode/tsconfig.json" },
```

Comment:
Not sure why it failed in this PR. But `dashboard` is importing types from `screenshot_mode` and it was missing this line.

@elastic/kibana-operations probably a bug in the `type_check` script that didn't catch it earlier in `main`?

Reply:
We're not totally sure how TypeScript validates that projects have the references they need, it seems to validate it most of the time but there are edge cases where you can import from another project even when it's not listed in the projects deps. We are on the cusp of moving away from relative plugin imports to imported based on global module IDs for packages which will make this a non-problem.

Reply:
This is a strange situation, but I'm glad we'll be moving away from it in the future. Thanks for adding the project to our tsconfig though!